Applications/CMC/cmc.cpp

```diff
@@ -111,20 +116,36 @@ int main(int argc,char **argv)
 
     // CONSTRUCT
     cout<<"Constructing investigation from setup file "<<setupFileName<<".\n"<<endl;
-    CMCTool cmcgait(setupFileName);
-
-    // PRINT MODEL INFORMATION
-    Model& model = cmcgait.getModel();
-    cout<<"-----------------------------------------------------------------------\n";
-    cout<<"Loaded library\n";
-    cout<<"-----------------------------------------------------------------------\n";
-    model.printBasicInfo(cout);
-    cout<<"-----------------------------------------------------------------------\n\n";
-
-
-    // RUN
-    cmcgait.run();
-
+    if(specifiedMaxNumThreads != -1)
+    {
+        CMCTool cmcgait(setupFileName,specifiedMaxNumThreads);
+        
```

Comment:
Can I only declare the CMCTool inside of the `if`? (Wouldn't it go out of scope?) Also, by default, if `specifiedMaxNumThreads` isn't set, it uses the multi-threaded implementation.